src/styles/utils/variables.scss

```diff
@@ -0,0 +1,7 @@
+$bodyFont: Roboto, sans-serif;
+$bodyColor: rgba(6, 11, 53, 1);
+$bodyColorHover: #34568b;
+$buttonColor: rgba(0, 172, 220, 1);
+$buttonTextColor: #fff;
+$active: #00acdc;
+$linkTextColor: #060b35;
```

Comment:
Consider using a consistent color format for your variables. You are using both `rgba` and hexadecimal formats. It's a good practice to stick to one format for color values for consistency. If you need transparency, use `rgba`, otherwise, hexadecimal is preferred for its simplicity.